FWCore/Utilities/interface/isFinite.h

```diff
@@ -32,7 +32,7 @@ namespace edm {
 
   template <>
   constexpr bool isFinite(long double x) {
-    double xx = x;
+    double xx = static_cast<double>(x);
     return isFinite(xx);
```

Comment:
I wonder if this `long double` overload really works properly (in presence of fast math), but ok, whatever the behavior is, it is there already.